pydis_site/apps/api/tests/test_off_topic_channel_names.py

```diff
@@ -10,12 +10,14 @@ def setUp(self):
         self.client.force_authenticate(user=None)
 
     def test_cannot_read_off_topic_channel_name_list(self):
+        """Test does this return 401 response code when not authenticated."""
```

Comment:
It's redundant to write "test" in the docstring of a test function. It's also awkward to use "does". If I were to use "test", I'd write it as 

> Test **if** this return**s** **a** 401 response when not authenticated.

Can also use "that" instead of "if". Anyway, as I said, I'd really opt for something along the lines of

> Should return a 401 response code when not authenticated.

 or

> Returned a 401 response code because of no authentication.

Reply:
I sort of agree with @MarkKoz, although docstrings should be imperative mood - so something like `Return a 401 response when not authenticated` seems more in line with PEP257 to me.